RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java

```diff
@@ -906,6 +906,158 @@ public void testAndNot_MultipleBucket() {
     assertEquals(123, left.select(0));
   }
 
+  @Test
+  public void testFlip_SameContainer() {
```

Comment:
please remove all the underline from the test method names

Reply:
done